docs/Contribute/ContributeWrite/README.md

```diff
@@ -1,32 +1,52 @@
 
-**Important note**: Contributions explained in this documentation are meant to be done in **English** only. If you wish to contribute in other languages, see [Translations](../ContributeTranslate.md)
+If you have writing skills or if you have a fair knowledge of the English language, then you can contribute to BTCPay Server or review the work of other contributors. Writing contributors can help in a number of places.
 
-## Requirements and recommended software
+## BTCPay Documentation
 
-### Requirements
+Documentation work needing to be completed can be found in the [open issues page](https://github.com/btcpayserver/btcpayserver-doc/issues) on Github. For information on how to start a documentation task, see the [documentation guide](WriteDocs.md). 
+
+For documentation guidance or discussion of a new documentation idea, join the [#documentation channel on Mattermost](https://chat.btcpayserver.org/btcpayserver/channels/documentation). 
+
+## BTCPay Blog
+
+Another great way to contribute is by writing content for the [BTCPay blog](WriteBlog.md). To get started on a blog topic, join the [#content creation channel on Mattermost](https://chat.btcpayserver.org/btcpayserver/channels/content-creation). 
+
+## Requirements
 
 In order to contribute to BTCPay Server as a writer there are some requirements.
 
-Since most of the writen done in BTCPay Server is done through Github, you must have a [Github account](https://github.com/).
-It's also on Github that you can review the work done by others.
+Since most of the documentation done in BTCPay Server is submitted and reviewed through Github, you must have a [Github account](https://github.com/). It's also on Github that you can contribute by reviewing the documentation work done by others (pull requests).
 
-### Recommended software
+**Important note**: Contributions explained in this documentation are meant to be done in **English** only. If you wish to contribute in other languages, see [Translations](../ContributeTranslate.md).
 
-To be able to fork repositories, create and work on branches, make Pull Requets and Issues easily, in a word, Manage your contributions, it's recommended your have [Github Desktop](https://desktop.github.com/).
+## Recommended software
 
-It is also recommended to have a Rich-Text editor.
+### Text Editor
+
+It is recommended to have a Rich-Text editor.
 
 [Notepad++](https://notepad-plus-plus.org/downloads/) is a decent software and easy to use for the newer contributors.
 
 For more advanced users, [Visual Studio Code (VS Code)](https://visualstudio.microsoft.com/) is a good choice.
-The `Markdown All In One` extension in VS Code is also recommended, for visual comfort while editing and the ability to preview Markdown changes.
+The `Markdown All In One` extension in VS Code is also recommended, for visual comfort while editing and the ability to preview Markdown changes. This is helpful for the BTCPay documentation which is written primarily in Markdown documentation.
```

Comment:
```suggestion
The `Markdown All In One` extension in VS Code is also recommended, for visual comfort while editing and the ability to preview Markdown changes. This is helpful for the BTCPay documentation which is written primarily in Markdown.
```

I don't know if we can call Markdown a `Markdown format`. If it's the case, go for it.
Otherwise, this deletes the word `documentation` which feels redundant.